<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>solicitudWidget</class>
 <widget class="QWidget" name="solicitudWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btnRechazar">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(239, 41, 41);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Rechazar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLabel" name="nombreCliente">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>0</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Quicksand Medium&quot;;
color: rgb(255, 255, 255);
background-color: rgb(215, 180, 254);</string>
   </property>
   <property name="text">
    <string>Nombre del cliente</string>
   </property>
  </widget>
  <widget class="QLabel" name="direccion">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>22</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(215, 180, 254);</string>
   </property>
   <property name="text">
    <string>Direccion</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTransp">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
   <item>
    <property name="text">
     <string>Seleccione un transportista</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="estatus">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Quicksand Medium&quot;;
border: 2px solid white;
border-radius: 5px;
background-color: rgb(215, 180, 254);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Estado</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="monto">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>40</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(215, 180, 254);</string>
   </property>
   <property name="text">
    <string>Monto</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(215, 180, 254);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>btnRechazar</zorder>
  <zorder>btnAceptar</zorder>
  <zorder>nombreCliente</zorder>
  <zorder>direccion</zorder>
  <zorder>comboBoxTransp</zorder>
  <zorder>estatus</zorder>
  <zorder>monto</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
